### PR TITLE
CERESTATION DOES NOT EXIST, COMRADE.

### PR DESCRIPTION
**Pull request by praisenarsie**
CERESTATION DOES NOT EXIST, COMRADE. 😤 

also how did we fucking miss this, it went RIGHT PAST OUR NOSES!

### CODE REVIEW

**Comment by Supermichael777** [comment]
Imgur

**Comment by ShizCalev** [comment]
https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/modules/paperwork/paper_premade.dm#L56

https://github.com/tgstation/tgstation/blob/af4d9a85c9a3eeef1e5e0637ef4ebd0312794ac7/code/game/objects/structures/signs.dm#L135

can probably remove these too.

File: code/modules/paperwork/paper_premade.dm [hunk 1]
```diff
@@ -53,43 +53,6 @@
 	*	Stations
 	*/
 
-//////////	Cere fluff
```
Comment (Cyberboss) [inline code/modules/paperwork/paper_premade.dm]: This sort of shit should be in the map files themselves imo. like map_files/Cerestation/fluff.dm

**Comment by FantasticFwoosh** [comment]
The ministry of factual information and lore demands that these changes so we can pursue our doublethink campaign to affirm that box and metastation are the only maps the game has to offer.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**